blockchain/blockchain.go

```diff
@@ -1614,10 +1616,20 @@ func (bc *BlockChain) insertChain(chain types.Blocks) (int, []interface{}, []*ty
 			return i, events, coalescedLogs, err
 		}
 
-		switch status {
+		switch writeResult.Status {
 		case CanonStatTy:
-			logger.Debug("Inserted new block", "number", block.Number(), "hash", block.Hash(),
-				"txs", len(block.Transactions()), "gas", block.GasUsed(), "elapsed", common.PrettyDuration(time.Since(bstart)))
+			localLogger := logger.Info
+			if bc.disableBlockSummary {
+				localLogger = logger.Debug
+			}
```

Comment:
How about setting the log level to debug by default or combining the following information in the report log, "Imported new chain segment ~ "? 
This change may confuse node operators since similar logs are printed twice.

Reply:
I think it's okay to combine it with the existing log.

Reply:
Maybe we can skip leaving a log when insertChain is called with one block, which usually means the node is at the latest state. How do you think? @aidan-kwon @KimKyungup

Reply:
How about printing the detail log if the elapsed time is bigger than 250 ms?